pkg/mcp/README.md

```diff
@@ -0,0 +1,304 @@
+# Openchoreo MCP Server
```

Comment:
Do we need this impl detail here? I think some parts, like configurations, we can move to the docs

Reply:
Moved the doc into the `docs/` directory by splitting into 2 files: configuring the mcp server and adding new tools to the mcp server (development guide)